packages/pegjs/lib/compiler/passes/generate-bytecode.js

```diff
@@ -26,7 +26,11 @@ const util = require( "../../util" );
 //
 //        stack.push(FAILED);
 //
-//  [4] PUSH_EMPTY_ARRAY
+//  [42] PUSH_FATAL
+//      
+//        throw e;
+// 
+//  [5] PUSH_EMPTY_ARRAY
```

Comment:
Why the number change?

Reply:
it's an incorrect by-hand tracking of the repeated data from opcodes.js

this should be brought from a documentation extractor.  there's several other mistakes in here too (such as several opcodes 5 and no opcode 4)